db/data/20240716232716_create_revision_reasons.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CreateRevisionReasons < ActiveRecord::Migration[7.1]
+  def up
+    site_configuration = SiteConfiguration.instance
+
+    site_configuration.revision_reasons.create(
+      [
+        { reason_code: "zoning_non_compliance", description: "Zoning non-compliance" },
+        { reason_code: "inaccurate_documentation", description: "Inaccurate documentation" },
+        { reason_code: "failure_to_meet_building_code", description: "Failure to meet building code requirements" },
+        { reason_code: "other", description: "Other" },
+      ],
+    )
```

Comment:
This should probably be a part of seeds.

Reply:
Are seeds populated on production? This is for production.

Reply:
@bkmaibach double check with Sunny but I think this should go into production seeds